scripts/archivist/htmlGeneration.js

```diff
@@ -23,15 +25,43 @@ Archivist.html.generateMetadataLabel = metadataField =>
       htmlFor: metadataField.id,
     });
 
-// Generates html for metadata field's input field
-Archivist.html.generateMetadataInput = (metadataField, groupName) => $('<input />',
-  {
-    // ID currently has spaces in it, which is no good for html ids
-    id: `${groupName.toLowerCase()}_${metadataField.id.replace(/ /i, '_')}`,
-    name: metadataField.name,
-    type: Archivist.html.util.getInputType(metadataField.type),
+Archivist.html.renderSelectOption = (optionArray, metadataField, groupName) => {
+  const select = $('<select></select>',
+    {
+      id: `${groupName.toLowerCase()}_${metadataField.id.replace(/ /i, '_')}`,
+    });
+
+  optionArray.forEach((option) => {
+    select.prepend(
+      $(`<option>${option.label}</option>`,
+        {
+          value: option.value,
+        }));
   });
 
+  return (select);
+};
+
+// Generates html for metadata field's input field
+Archivist.html.generateMetadataInput = (metadataField, groupName) => {
+  if (metadataField.type === 'true/false') {
```

Comment:
I set up the types in the backend to be boolean for us, but true/false to the user on display. In this case the api only returns the frontend type I believe, I will double check though.